### PR TITLE
Fixed ERD table and column naming convention

### LINKED ISSUE #45: [Bug] Fix ERD Table Naming Convention
Fix ERD Table Naming Convention

## Task Description
- Change the naming convention of tables from camel case to underscores because that's what postgresql prefers (which is the database system we're using).
## User Stories
- As a developer using postgresql, I'd like my ERD to follow the correct naming conventions.
## Relevant Files [If Available]
- README.md

### PR DESCRIPTION
**Pull request by isaacschaessenscoleman**
Along with @adamski201 fix to the naming conventions (camel case -> snake case) in the database/schema.sql file (PR #47), I've accordingly changed the ERD in our README.md to match those changes exactly.

Closes #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,9 +73,10 @@ terraform destroy
 
 ### ERD diagram
 
-[ERD diagram](https://drawsql.app/teams/trainee-4/diagrams/railway-tracker-3nf)
+[ERD diagram](https://drawsql.app/teams/isaac-11/diagrams/railwaytracker-erd)
```
Comment (SaniyaShaikhh) [inline README.md:76]: I don't think we need to include the drawsql link here, I think just the image is sufficient.

**Review by adamski201 — COMMENTED** [review]
Looks good! As Saniya mentioned we could remove the drawsql link as we are finished with that.

File: README.md (hunk 1 above)
Comment (isaacschaessenscoleman) [reply README.md]: The ERD link has been removed.

**Review by aribasyeda — APPROVED** [review]
Great work Isaac 🥇

**Pull request merged**